src/Providers/UserInterfaceServiceProvider.php

```diff
@@ -207,6 +207,9 @@ private function registerPublishers(): void
     private function registerBladeComponents(): void
     {
         $this->callAfterResolving(BladeCompiler::class, function (BladeCompiler $blade) {
+            // Load every `<x-ark:: />` component from the `ark::` View namespaces...
+            $blade->anonymousComponentNamespace('ark::', 'ark');
```

Comment:
if it loads everything, can all the explicit components be removed?

Reply:
Yeah, but it would be a huge change.

This now allows us to reference components from the ark namespace, for example `<x-ark::checkbox />`. Since we explicitly define components we can use them as `<x-ark-checkbox />`, so removing them would require us to update every project's `<x-ark-component` to `x-ark::component`.

I think it would be better if we could just slowly do that, component by component, so it's not one giant change all at once.